_posts/2019-12-16-event-store-6.0.0-preview-release.md

```diff
@@ -0,0 +1,386 @@
+
+Much of the platform-specific code has been removed, and we benefit from the focus on performance that Microsoft has given the Core CLR.   We now use the Kestrel HTTP server, which has significant benefits over HttpListener which was used up to version 5.
+
+Historically we have provided Linux and macOS binaries with the Mono runtime statically linked into them. Unfortunately this is not yet supported by .NET Core. Consequently we now ship the correct version of the runtime as part of our packages.
+
+## gRPC Client API
+
+Since version 1.0 of Event Store, two client protocols have been supported - Atompub over HTTP, and full-duplex protocol-buffers-over-TCP.
+
+Each of these has advantages - the Protocol Buffers API is better for latency-sensitive workloads, while the Atom API gives more flexibility for cache design. However, both require a lot of work on each platform to implement.
+
+To improve this situation, we have added a new default client protocol in Event Store 6, using gRPC - a widely adopted standard which is interoperable across lots of platforms with minimal effort.
+
+In Preview 1 of Event Store 6, we are shipping a .NET SDK for the new gRPC protocol. With the final version of Event Store 6, we will also ship first party, commercially supported client SDKs for Java, Node.js and Go in addition to .NET.
+
+The Event Store .NET gRPC SDK is available on [NuGet](https://www.nuget.org/packages/EventStore.Grpc.Client/).
+
+The following code snippets show how to perform some common operations with the .NET SDK:
+
+### Create the Client
+
+```c#
+var eventStoreClient = new EventStore.Grpc.EventStoreGrpcClient(new Uri("https://localhost:2113/"));
+```
+
+**Note:**  
+In order for the gRPC client to connect successfully, the TLS certificate used by Event Store must be trusted by your machine.  
+When running in development mode, this would be the development certificates found in `dev-ca` in the extracted directory.
+
+Alternatively, you can disable the validation checks by specifying a custom HttpClient when creating the client, for example:
+
+```c#
+var eventStoreClient = new EventStore.Grpc.EventStoreGrpcClient(new Uri("https://localhost:2113/"),
+    () => new HttpClient(new SocketsHttpHandler
+    {
+        SslOptions = new SslClientAuthenticationOptions
+        {
+            RemoteCertificateValidationCallback = delegate { return true; }
+        }
+    }));
+```
+
+### Writing to a Stream with Expected Version Any
+
+```c#
+var eventData = new EventData(Uuid.NewUuid(), "type", JsonSerializer.SerializeToUtf8Bytes(new SomeEvent()));
+await eventStoreClient.AppendToStreamAsync(
+  "stream", AnyStreamRevision.Any, new[] { eventData }, cancellationToken: cancellationToken);
+```
+
+### Writing to a Stream with Expected Version
+
+```c#
+var eventData = new EventData(Uuid.NewUuid(), "type", JsonSerializer.SerializeToUtf8Bytes(new SomeEvent()));
```

Comment:
One suggestion, I think it's better to use  the full type name here i.e. `System.Text.Json.JsonSerializer.SerializeToUtf8Bytes`, otherwise there might be confusion as `Newtonsoft.Json` has the same name here.